tests/spread/core24/python-hello/task.yaml

```diff
@@ -0,0 +1,20 @@
+summary: Build and run Python-based snaps in core24
+
+environment:
+  CONFINEMENT/strict: strict
+  CONFINEMENT/classic: classic
+
+restore: |
+  cd ./"${CONFINEMENT}"
+  snapcraft clean
+  rm -f ./*.snap
+  snap remove python-hello-"${CONFINEMENT}"
```

Comment:
this is not needed due to https://github.com/canonical/snapcraft/blob/feature/craft-application/spread.yaml#L225 which calls https://github.com/canonical/snapcraft/blob/057e21b7a20bf698b7d94a9fd9ed65e00919254c/tests/spread/tools/restore.sh